src/js/endpoint.js

```diff
@@ -6,22 +6,27 @@ module.exports.rootUrl = ROOT_URL;
 
 // State params set by main
 module.exports.appToken = undefined;
+module.exports.jwt = undefined;
 module.exports.appUserId = undefined;
 
 module.exports._rest = function(method, path, body) {
     var deferred = $.Deferred();
+    var headers = {
+        'app-token': this.appToken
+    };
+
+    this.jwt ? headers.Authorization = 'Bearer ' + this.jwt : '';
```

Comment:
Yeah whenever I'm in a situation where I want to do something like that it takes me like 5 minutes to decide one way or the other.
I wish there was an operator that allowed it.